introduction_to_applying_machine_learning/gluon_recommender_system/gluon_recommender_system.ipynb

```diff
@@ -933,9 +880,7 @@
   {
```

Comment:
This model is producing a perfect correlation between multiple user 6 and user 7's reviews.

Reply:
Thanks a lot for the review! I added the changes.